### ISSUE TITLE
Result of BCE without kl divergence

### ISSUE THREAD

**Issue by selous123** [YisenWang/MART#6]
Hi, thanks for your promising work.

We have run your code with \beta equals 0

we got the result as follows:

```
best robust acc:0.5485000014305115, natural acc:0.8273999691009521 @epoch:80
```

which is much superior to the reported result in figure2(a)@paper [best is almost around 51% in your figure]

**Is \beta = 0 equals the figure2(a)@BCE(x_adv)?**

**Comment by YisenWang** [comment]
Yes,  \beta = 0 equals the figure2(a)@bce(x_adv). But I guess your results are from different parameters: weight decay and learning rate.

**Comment by selous123** [comment]
Thank you for your reply.

i have re-read your paper and notice the difference of experiment settings between section 4.1 and section 4.2.

i am curious about why the difference of the setting exisits.

**Comment by YisenWang** [comment]
Setting in 4.2 strikes a better tradeoff between clean acc and robustness, with slightly increased clean acc. I suggest you use the setting in 4.1.
